composer/packages/documentation/src/components/DocPreview.jsx

```diff
@@ -146,6 +146,10 @@ export default class DocPreview extends React.Component {
                 console.error(e);
             }
         });
-        return docElements;
+        if (docElements.length) {
+            return docElements;
+        } else {
+            return <p>{"No documentations to show"}</p>
```

Comment:
Better to put "documentation" instead of documentations?

Reply:
yeah that's better. updated.